intel_extension_for_transformers/llm/operator/cscr/run_build.sh

```diff
@@ -11,10 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-rm -rf build
-mkdir build
+rm -rf build;
+mkdir build;
 cd build
 
-cmake ..
+cmake  ..
```

Comment:
Add CMAKE_BUILD_TYPE here instead of adding hard-code CXX_FLAGS in CMakeLists.txt